packages/venia-ui/lib/queries/getCategoryList.graphql

```diff
@@ -15,6 +15,7 @@ query categoryList($id: Int!) {
                     small_image {
                         url
                     }
+                    url_key
```

Comment:
I'd rather we not fetch the key here I think. We aren't actually using that piece of data where we use this query and that feels counter to the intent of the granularity of graphql.

Reply:
https://github.com/magento/pwa-studio/pull/1935/commits/bf4b0dd741eddc8ed586fffbea6245814fd7b94d.